CONTRIBUTING.md

```diff
@@ -14,6 +14,8 @@ If you have a local WordPress environment, you can clone this repository into yo
 
 Some good options for a local WordPress development environment include <a href="https://varyingvagrantvagrants.org/">VVV</a> and <a href="https://www.mamp.info/">Mamp</a>.
 
+> Once the plugin is intalled, and you have built it through `npm` go to the post list and hover over a post row to click `Edit in Gutenberg` link. That should do it for initial testing.
```

Comment:
Should we mention the admin bar menu which includes both a Demo and New Post submenus?

Nitpicking, but the last sentence doesn't add much value to the text, and could be dropped without sacrificing clarity.